libs/output-mapping/tests/Writer/Workspace/WriterWorkspaceTest.php

```diff
@@ -591,6 +591,52 @@ public function testWriteOnlyOnJobFailure(): void
         ]);
     }
 
+    #[NeedsTestTables(2), NeedsEmptyOutputBucket]
+    public function testOrphanedManifest(): void
```

Comment:
tak on jako není orphaned, když to je expected :) testManifestOnly nebo testManifestWithoutConfiguration ?